src/site/content/en/blog/file-system-access/index.md

```diff
@@ -210,83 +215,147 @@ async function writeURLToFile(fileHandle, url) {
 
 ```js
 const butDir = document.getElementById('butDirectory');
-butDir.addEventListener('click', async (e) => {
-  const opts = {type: 'open-directory'};
-  const handle = await window.chooseFileSystemEntries(opts);
-  const entries = await handle.getEntries();
-  for await (const entry of entries) {
-    const kind = entry.isFile ? 'File' : 'Directory';
-    console.log(kind, entry.name);
+butDir.addEventListener('click', async () => {
+  const dirHandle = await window.showDirectoryPicker();
+  for await (const entry of dirHandle.values()) {
+    console.log(entry.kind, entry.name);
   }
 });
 ```
 
+### Creating or accessing files and folders in a directory
+
+From a directory, you can create or access files and folders using the
+[`getFileHandle()`][getfilehandle] or respectively the [`getDirectoryHandle()`][getdirectoryhandle]
+method. By passing in an optional `options` object with a key of `create` and a boolean value of
+`true` or `false`, you can determine if a new file or folder should be created if it doesn't exist.
+
+```js
+// In an existing directory, create a new directory named "My Documents".
+const newDirectoryHandle = await existingDirectoryHandle.getDirectoryHandle('My Documents', {
+  create: true,
+});
+// In this new directory, create a file named "My Notes.txt".
+const newFileHandle = await newDirectoryHandle.getFileHandle('My Notes.txt', { create: true });
+```
+
+### Resolving the path of an item in a directory
+
+When working with files or folders in a directory, it can be useful to resolve the path of the item
+in question. This can be done with the aptly named [`resolve()`][resolve] method. For resolving, the
+item can be a direct or indirect child of the directory.
+
+```js
+// Resolve the path of the previously created file called "My Notes.txt".
+const path = await newDirectoryHandle.resolve(newFileHandle);
+// `path` is now ["My Documents", "My Notes.txt"]
+```
+
+### Deleting files and folders in a directory
+
+If you have obtained access to a directory, you can delete the contained files and folders with the
+[`removeEntry()`][removeentry] method, for folders optionally recursively including all subfolders
+and the therein contained files.
+
+```js
+// Delete a file.
+await directoryHandle.removeEntry('Abandoned Masterplan.txt');
+// Recursively delete a folder.
+await directoryHandle.removeEntry('Old Stuff', { recursive: true });
+```
+
+### Accessing the origin-private file system
+
+The origin-private file system is a storage endpoint that, as the name suggests, is private to the
+origin of the page. While browsers will typically implement this by persisting the contents of this
+origin-private file system to disk somewhere, it is _not_ intended that the contents be easily user
+accessible. Similarly, there is _no_ expectation that files or directories with names matching the
+names of children of the origin-private file system exist. You can operate as usual on the
```

Comment:
What this is saying is that while the browser might make it *seem* that there are files, internally—since this is an origin-private file system—the browser might store these "files" in a database or any other data structure. Essentially: what you create with this API, don't expect to find it 1:1 somewhere on the hard disk.

Reply:
Then please say it the way you just said it me and we'll be done.